README.md

```diff
@@ -54,7 +54,8 @@ documentation](https://libvirt.org/formatdomain.html#elementsCPU).
      aytests-tests):
 
         $ zypper in ruby2.1-rubygem-aytests
-        $ git clone https://github.com/yast/aytests-tests
+        $ git clone https://github.com/yast/autoyast-integration-test.git
```

Comment:
This line refers to the tests themselves so the correct repository is https://github.com/yast/aytests-tests.